Sketch.java

```diff
@@ -16,21 +20,139 @@ public void settings() {
    * values here i.e background, stroke, fill etc.
    */
   public void setup() {
-    background(210, 255, 173);
+    background(173, 230, 255);
   }
 
   /**
    * Called repeatedly, anything drawn to the screen goes here
    */
   public void draw() {
 	  
-	// sample code, delete this stuff
-    stroke(128);
-    line(150, 25, 270, 350);  
+    // Draw body
```

Comment:
missing [level 3 requirement](https://github.com/SACHSTech/ProcessingTask6-CreatingMethods#level-3), "draw a composite object at various locations specified by the method parameters."